test/contracts/ethereumRuntime.js

```diff
@@ -108,7 +113,7 @@ describe('Runtime', function () {
 
         if (gasUsedBaseline !== undefined) {
           // The max increase in gas usage
-          const maxAllowedDiff = 5000;
+          const maxAllowedDiff = 500000;
```

Comment:
maybe create an issue to kick this out ASAP?

Reply:
https://github.com/leapdao/solEVM-enforcer/issues/171